pkgs/development/python-modules/tensorflow/default.nix

```diff
@@ -369,13 +374,14 @@ let
     fetchAttrs = {
       # cudaSupport causes fetch of ncclArchive, resulting in different hashes
       sha256 = if cudaSupport then
-        "sha256-KtVReqHL3zxE8TPrqIerSOt59Mgke/ftoFZKMzgX/u8="
+        "sha256-SudzMTxfifKJJso6haCgOD2dXeAhYSXHA2nzq1ErTHg="
+      else if stdenv.isDarwin && stdenv.isAarch64 then
+        # FIXME: this checksum is currently wrong, since the tensorflow dependency fetch is broken on darwin
+        "sha256-u+ODHAZDlGe06PUWId4sNKyl60vhAPMd01jMm2EvN8E="
+      else if stdenv.isDarwin && stdenv.isx86_64 then
+        "sha256-j2k9Q+k41444nP1VjjkkNjXRov1uAda4RCMDMAthjr0="
```

Comment:
```suggestion
        "sha256-/qPUDgfKsWCZh/pgZM4wm9+4U9U5kxxv7q3Uh7zKSO4="
```

Reply:
Yes, this was the first issue I ran into as well but on aarch64-darwin. Since I don't have any x86-darwin machine, could you retest with the suggested change?